src/app/loadout-builder/process-worker/auto-stat-mod-utils.ts

```diff
@@ -8,22 +8,22 @@ import { ProcessMod } from './types';
 const largeStatMods: {
   [statHash in ArmorStatHashes]: { hash: number; cost: number; statHash: ArmorStatHashes };
 } = {
-  [StatHashes.Mobility]: { hash: 3961599962, cost: 3, statHash: StatHashes.Mobility },
-  [StatHashes.Resilience]: { hash: 2850583378, cost: 3, statHash: StatHashes.Resilience },
-  [StatHashes.Recovery]: { hash: 2645858828, cost: 4, statHash: StatHashes.Recovery },
-  [StatHashes.Discipline]: { hash: 4048838440, cost: 3, statHash: StatHashes.Discipline },
-  [StatHashes.Intellect]: { hash: 3355995799, cost: 5, statHash: StatHashes.Intellect },
-  [StatHashes.Strength]: { hash: 3253038666, cost: 3, statHash: StatHashes.Strength },
+  [StatHashes.Mobility]: { hash: 4183296050, cost: 3, statHash: StatHashes.Mobility },
+  [StatHashes.Resilience]: { hash: 1180408010, cost: 4, statHash: StatHashes.Resilience },
+  [StatHashes.Recovery]: { hash: 4204488676, cost: 4, statHash: StatHashes.Recovery },
+  [StatHashes.Discipline]: { hash: 1435557120, cost: 3, statHash: StatHashes.Discipline },
+  [StatHashes.Intellect]: { hash: 2724608735, cost: 4, statHash: StatHashes.Intellect },
+  [StatHashes.Strength]: { hash: 4287799666, cost: 3, statHash: StatHashes.Strength },
```

Comment:
I guess we should generate these from d2ai now?

Reply:
Once I sort out the rest of LO yes, this is just to fix the crash with the auto mods toggle